lib/iris/tests/test_image_json.py

```diff
@@ -34,32 +34,47 @@
 
 
 @tests.skip_inet
+@tests.skip_data
 class TestImageFile(tests.IrisTest):
     def test_resolve(self):
-
-        iuri = ('https://api.github.com/repos/scitools/test-iris-imagehash/'
-                'contents/images')
         # https://developer.github.com/v3/#user-agent-required
-        headers = {'User-Agent': 'SciTools'}
-        r = requests.get(iuri, headers=headers)
-        if r.status_code != 200:
-            raise ValueError('Github API get failed: {}'.format(iuri))
-        rj = r.json()
-        prefix = 'https://scitools.github.io/test-iris-imagehash/images/'
+        headers = {'User-Agent': 'scitools-bot'}
+        rate_limit_uri = 'https://api.github.com/rate_limit'
+        rl = requests.get(rate_limit_uri, headers=headers)
+        some_left = False
+        if rl.status_code == 200:
+            rates = rl.json()
+            remaining = rates.get('rate', {})
+            remaining = remaining.get('remaining')
+            if isinstance(remaining, int):
+                if remaining > 3:
```

Comment:
i thought about >0 but then there's a race condition between the python2 tests and the python3 tests

so, 3 is an engineering tolerance